docs/docs/tutorial/part-4/index.mdx

```diff
@@ -0,0 +1,918 @@
+  heading,
+  navLinks,
+  navLinkItem,
+  navLinkText
+} from './layout.module.css'
+
+const Layout = ({ pageTitle, children }) => {
+  const data = useStaticQuery(graphql`
+    query LayoutQuery {
+      site {
+        siteMetadata {
+          title
+        }
+      }
+    }  
+  `)
+  
+  return (
+    <main className={container}>
+      {/* highlight-start */}
+      <title>{pageTitle} | {data.site.siteMetadata.title}</title>
+      <p>{data.site.siteMetadata.title}</p>
+      {/* highlight-end */}
+      <nav>
+        <ul className={navLinks}>
+          <li className={navLinkItem}>
+            <Link to="/" className={navLinkText}>
+              Home
+            </Link>
+          </li>
+          <li className={navLinkItem}>
+            <Link to="/about" className={navLinkText}>
+              About
+            </Link>
+          </li>
+        </ul>
+      </nav>
+      <h1 className={heading}>{pageTitle}</h1>
+      {children}
+    </main>
+  )
+}
+
+export default Layout
+```
+
+![A screenshot of the home page in a web browser. The title of the page in the browser tab now says, "Home Page | My First Gatsby Site", and the top of the page has an unstyled paragraph that says, "My First Gatsby Site".](./index-page-with-site-title.png)
+
+4. Now that the site title is showing up on the page, it's time to add some style! Define some styles for the site title below the existing styles in your `layout.module.css` file.
+
+```css:title=src/components/layout.module.css
+/* ... your existing styles */
+
+.site-title {
+  font-size: 3rem;
+  color: gray;
+  font-weight: 700;
+}
+```
+
+5. Import your new styles into your `Layout` component and apply them to the site title paragraph you added.
+
+```js:title=src/components/layout.js
+import * as React from 'react'
+import { Link, useStaticQuery, graphql } from 'gatsby'
+import {
+  container,
+  heading,
+  navLinks,
+  navLinkItem,
+  navLinkText,
+  siteTitle, // highlight-line
+} from './layout.module.css'
+
+const Layout = ({ pageTitle, children }) => {
+  const data = useStaticQuery(graphql`
+    query LayoutQuery {
+      site {
+        siteMetadata {
+          title
+        }
+      }
+    }  
+  `)
+  
+  return (
+    <main className={container}>
+      <title>{pageTitle} | {data.site.siteMetadata.title}</title>
+      {/* highlight-next-line */}
+      <p className={siteTitle}>{data.site.siteMetadata.title}</p>
+      <nav>
+        <ul className={navLinks}>
+          <li className={navLinkItem}>
+            <Link to="/" className={navLinkText}>
+              Home
+            </Link>
+          </li>
+          <li className={navLinkItem}>
+            <Link to="/about" className={navLinkText}>
+              About
+            </Link>
+          </li>
+        </ul>
+      </nav>
+      <h1 className={heading}>{pageTitle}</h1>
+      {children}
+    </main>
+  )
+}
+
+export default Layout
+```
+
+Reload your browser to see your new styles applied to your site title.
```

Comment:
In theory this should hot-reload?

Reply:
Good point. Tweaked the wording to hopefully convey that it should happen automatically.

```suggestion
When your browser reloads, you should see your new styles applied to your site title.
```